cfy_manager/utils/service.py

```diff
@@ -202,13 +238,27 @@ def is_alive(self, service_name):
             'status', service_name, ignore_failure=True)
         return result.returncode == 0
 
+    def is_active(self, service_name):
+        response = self.supervisorctl(
+            'status', service_name,
+            ignore_failure=True
+        )
+        # Output of `supervisorctl -c /etc/supervisord.conf status SERVICE`
+        # is on the following format
+        # "SERVICE                          EXITED    Jun 09 09:35 AM"
+        # Remove new line and then split the output to get the desired status
+        status = response.aggr_stdout.strip().split()[1]
```

Comment:
This could probably be rewritten to be one instruction, but keeping the comment above it?
(rewritten in a similar way to the other is_active)

Reply:
Done